### PR TITLE
Fix/pr storing

### LINKED ISSUE #389: Kebechet metrics calculation fails due to the `id` error
**Describe the bug**
SSIA

**LOGS**
```
INFO:srcopsmetrics.github_knowledge:Overall repositories found: 1
INFO:srcopsmetrics.bot_knowledge:######################## Analysing thoth-station/stub-api ########################

INFO:srcopsmetrics.utils:No repo identified, creating new directory at /opt/app-root/src/srcopsmetrics/bot_knowledge/thoth-station/stub-api
INFO:srcopsmetrics.bot_knowledge:########################
INFO:srcopsmetrics.bot_knowledge:Detected entities:
CodeFrequency # Commit # DependencyUpdate # Fork # Issue # KebechetUpdateManager # PullRequest # PullRequestDiscussion # ReadMe # Release # Stargazer
INFO:srcopsmetrics.bot_knowledge:########################
INFO:srcopsmetrics.bot_knowledge:Issue inspection
INFO:srcopsmetrics.utils:No repo identified, creating new directory at /opt/app-root/src/thoth-test-core/thoth-sli-metrics/kebechet-update-manager/thoth-station/stub-api/bot_knowledge/thoth-station/stub-api
INFO:srcopsmetrics.entities.tools.storage:Loading knowledge from Ceph
INFO:srcopsmetrics.entities.interface:No previous knowledge of type Issue found
INFO:srcopsmetrics.iterator:-------------Issue Analysis-------------

  0%|          | 0/10 [00:00<?, ?it/s]
 10%|█         | 1/10 [00:00<00:03,  2.81it/s]
 20%|██        | 2/10 [00:00<00:02,  3.41it/s]
 30%|███       | 3/10 [00:00<00:01,  3.67it/s]
 40%|████      | 4/10 [00:01<00:01,  3.56it/s]INFO:srcopsmetrics.iterator:[API requests remaining: 4062]

                                              

 40%|████      | 4/10 [00:01<00:01,  3.56it/s]
 50%|█████     | 5/10 [00:01<00:01,  3.67it/s]
 60%|██████    | 6/10 [00:01<00:01,  3.58it/s]
 70%|███████   | 7/10 [00:01<00:00,  3.76it/s]
 80%|████████  | 8/10 [00:02<00:00,  4.04it/s]
 90%|█████████ | 9/10 [00:02<00:00,  4.04it/s]INFO:srcopsmetrics.iterator:[API requests remaining: 4052]

                                              

 90%|█████████ | 9/10 [00:02<00:00,  4.04it/s]
100%|██████████| 10/10 [00:02<00:00,  3.98it/s]
100%|██████████| 10/10 [00:02<00:00,  3.77it/s]
INFO:srcopsmetrics.entities.interface:Knowledge file Issue.json
INFO:srcopsmetrics.entities.interface:new 10 entities
INFO:srcopsmetrics.entities.interface:(overall 10 entities)
INFO:srcopsmetrics.entities.interface:Saved on CEPH at thoth/data/thoth-test-core/thoth-sli-metrics/kebechet-update-manager/thoth-station/stub-api/bot_knowledge/thoth-station/stub-api/Issue.json
INFO:srcopsmetrics.bot_knowledge:

INFO:srcopsmetrics.bot_knowledge:PullRequest inspection
INFO:srcopsmetrics.entities.tools.storage:Loading knowledge from Ceph
INFO:srcopsmetrics.entities.interface:No previous knowledge of type PullRequest found
INFO:srcopsmetrics.iterator:-------------PullRequest Analysis-------------

0it [00:00, ?it/s]INFO:srcopsmetrics.entities.pull_request:Extracting PR #96

                  

0it [00:00, ?it/s]
1it [00:02,  2.06s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #95

                      

1it [00:02,  2.06s/it]
2it [00:03,  1.56s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #94

                      

2it [00:03,  1.56s/it]INFO:srcopsmetrics.entities.pull_request:      -analysing review no. 1/1

                      

2it [00:04,  1.56s/it]
3it [00:04,  1.43s/it]
                      
INFO:srcopsmetrics.entities.pull_request:Extracting PR #93

3it [00:04,  1.43s/it]INFO:srcopsmetrics.entities.pull_request:      -analysing review no. 1/1

                      

3it [00:05,  1.43s/it]
4it [00:05,  1.42s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 4015]

                      

4it [00:05,  1.42s/it]
                      
INFO:srcopsmetrics.entities.pull_request:Extracting PR #92

4it [00:05,  1.42s/it]
5it [00:07,  1.38s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #91

                      

5it [00:07,  1.38s/it]
6it [00:08,  1.29s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #90

                      

6it [00:08,  1.29s/it]
7it [00:09,  1.22s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #89

                      

7it [00:09,  1.22s/it]
8it [00:10,  1.19s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #88

                      

8it [00:10,  1.19s/it]
9it [00:11,  1.16s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3974]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #87

                      

9it [00:11,  1.16s/it]
                      

9it [00:11,  1.16s/it]
10it [00:12,  1.18s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #86

                       

10it [00:12,  1.18s/it]
11it [00:14,  1.18s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #85

                       

11it [00:14,  1.18s/it]
12it [00:15,  1.16s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #83

                       

12it [00:15,  1.16s/it]
13it [00:16,  1.13s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #81

                       

13it [00:16,  1.13s/it]
14it [00:17,  1.15s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3934]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #80

                       

14it [00:17,  1.15s/it]
                       

14it [00:17,  1.15s/it]
15it [00:18,  1.13s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #79

                       

15it [00:18,  1.13s/it]
16it [00:19,  1.11s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #78

                       

16it [00:19,  1.11s/it]
17it [00:20,  1.12s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #77

                       

17it [00:20,  1.12s/it]
18it [00:21,  1.11s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #76

                       

18it [00:21,  1.11s/it]
19it [00:22,  1.11s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3894]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #75

                       

19it [00:22,  1.11s/it]
                       

19it [00:22,  1.11s/it]
20it [00:24,  1.10s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #74

                       

20it [00:24,  1.10s/it]
21it [00:25,  1.10s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #73

                       

21it [00:25,  1.10s/it]
22it [00:26,  1.09s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #71

                       

22it [00:26,  1.09s/it]
23it [00:27,  1.11s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #70

                       

23it [00:27,  1.11s/it]
24it [00:28,  1.09s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3854]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #68

                       

24it [00:28,  1.09s/it]
                       

24it [00:28,  1.09s/it]
25it [00:29,  1.09s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #67

                       

25it [00:29,  1.09s/it]
26it [00:30,  1.11s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #66

                       

26it [00:30,  1.11s/it]
27it [00:31,  1.08s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #65

                       

27it [00:31,  1.08s/it]
28it [00:32,  1.09s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #64

                       

28it [00:32,  1.09s/it]
29it [00:33,  1.11s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3814]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #63

                       

29it [00:33,  1.11s/it]
                       

29it [00:33,  1.11s/it]
30it [00:34,  1.11s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #62

                       

30it [00:35,  1.11s/it]
31it [00:37,  1.38s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #61

                       

31it [00:37,  1.38s/it]
32it [00:38,  1.27s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #60

                       

32it [00:38,  1.27s/it]INFO:srcopsmetrics.entities.pull_request:      -analysing review no. 1/1

                       

32it [00:38,  1.27s/it]
33it [00:39,  1.28s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #59

                       

33it [00:39,  1.28s/it]INFO:srcopsmetrics.entities.pull_request:      -analysing review no. 1/4
INFO:srcopsmetrics.entities.pull_request:      -analysing review no. 2/4
INFO:srcopsmetrics.entities.pull_request:      -analysing review no. 3/4
INFO:srcopsmetrics.entities.pull_request:      -analysing review no. 4/4

                       

33it [00:40,  1.28s/it]
                       

33it [00:40,  1.28s/it]
                       

33it [00:40,  1.28s/it]
                       

33it [00:40,  1.28s/it]
34it [00:40,  1.32s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3773]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #58

                       

34it [00:40,  1.32s/it]
                       

34it [00:40,  1.32s/it]
35it [00:42,  1.33s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #57

                       

35it [00:42,  1.33s/it]
36it [00:43,  1.41s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #56

                       

36it [00:43,  1.41s/it]
37it [00:44,  1.35s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #55

                       

37it [00:44,  1.35s/it]
38it [00:46,  1.42s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #54

                       

38it [00:46,  1.42s/it]INFO:srcopsmetrics.entities.pull_request:      -analysing review no. 1/1

                       

38it [00:47,  1.42s/it]
39it [00:48,  1.57s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3726]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #53

                       

39it [00:48,  1.57s/it]
                       

39it [00:48,  1.57s/it]
40it [00:49,  1.51s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #52

                       

40it [00:49,  1.51s/it]
41it [00:51,  1.54s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #51

                       

41it [00:51,  1.54s/it]
42it [00:52,  1.46s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #50

                       

42it [00:52,  1.46s/it]
43it [00:53,  1.37s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #49

                       

43it [00:53,  1.37s/it]
44it [00:55,  1.35s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3680]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #47

                       

44it [00:55,  1.35s/it]
                       

44it [00:55,  1.35s/it]
45it [00:56,  1.33s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #46

                       

45it [00:56,  1.33s/it]
46it [00:57,  1.30s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #45

                       

46it [00:57,  1.30s/it]
47it [00:58,  1.25s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #44

                       

47it [00:58,  1.25s/it]
48it [00:59,  1.21s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #43

                       

48it [00:59,  1.21s/it]
49it [01:01,  1.20s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3635]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #42

                       

49it [01:01,  1.20s/it]
                       

49it [01:01,  1.20s/it]
50it [01:02,  1.19s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #41

                       

50it [01:02,  1.19s/it]
51it [01:03,  1.25s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #40

                       

51it [01:03,  1.25s/it]
52it [01:04,  1.23s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #39

                       

52it [01:04,  1.23s/it]
53it [01:05,  1.22s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #38

                       

53it [01:06,  1.22s/it]
54it [01:07,  1.22s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3590]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #37

                       

54it [01:07,  1.22s/it]
                       

54it [01:07,  1.22s/it]
55it [01:08,  1.22s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #36

                       

55it [01:08,  1.22s/it]
56it [01:09,  1.23s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #35

                       

56it [01:09,  1.23s/it]
57it [01:10,  1.21s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #34

                       

57it [01:10,  1.21s/it]
58it [01:11,  1.18s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #33

                       

58it [01:11,  1.18s/it]
59it [01:13,  1.17s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3545]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #32

                       

59it [01:13,  1.17s/it]
                       

59it [01:13,  1.17s/it]
60it [01:14,  1.17s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #31

                       

60it [01:14,  1.17s/it]
61it [01:15,  1.32s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #30

                       

61it [01:15,  1.32s/it]
62it [01:17,  1.27s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #29

                       

62it [01:17,  1.27s/it]
63it [01:18,  1.24s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #28

                       

63it [01:18,  1.24s/it]
64it [01:19,  1.21s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3499]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #26

                       

64it [01:19,  1.21s/it]
                       

64it [01:19,  1.21s/it]
65it [01:20,  1.20s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #25

                       

65it [01:20,  1.20s/it]
66it [01:21,  1.23s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #24

                       

66it [01:21,  1.23s/it]
67it [01:23,  1.29s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #23

                       

67it [01:23,  1.29s/it]
68it [01:24,  1.29s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #22

                       

68it [01:24,  1.29s/it]
69it [01:26,  1.39s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3454]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #21

                       

69it [01:26,  1.39s/it]
                       

69it [01:26,  1.39s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

69it [01:26,  1.39s/it]
70it [01:27,  1.35s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #20

                       

70it [01:27,  1.35s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

70it [01:27,  1.35s/it]
71it [01:28,  1.30s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #19

                       

71it [01:28,  1.30s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

71it [01:29,  1.30s/it]
72it [01:29,  1.25s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #18

                       

72it [01:29,  1.25s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

72it [01:30,  1.25s/it]
73it [01:30,  1.22s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #17

                       

73it [01:31,  1.22s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

73it [01:31,  1.22s/it]
74it [01:32,  1.20s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3409]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #16

                       

74it [01:32,  1.20s/it]
                       

74it [01:32,  1.20s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

74it [01:32,  1.20s/it]
75it [01:33,  1.28s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #15

                       

75it [01:33,  1.28s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

75it [01:33,  1.28s/it]
76it [01:34,  1.24s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #14

                       

76it [01:34,  1.24s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

76it [01:35,  1.24s/it]
77it [01:35,  1.23s/it]
                       
INFO:srcopsmetrics.entities.pull_request:Extracting PR #13

77it [01:35,  1.23s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

77it [01:36,  1.23s/it]
78it [01:37,  1.20s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #12

                       

78it [01:37,  1.20s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

78it [01:37,  1.20s/it]
79it [01:38,  1.20s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3364]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #11

                       

79it [01:38,  1.20s/it]
                       

79it [01:38,  1.20s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

79it [01:38,  1.20s/it]
80it [01:39,  1.20s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #10

                       

80it [01:39,  1.20s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

80it [01:39,  1.20s/it]
81it [01:40,  1.17s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #9

                       

81it [01:40,  1.17s/it]ERROR:srcopsmetrics.entities.tools.knowledge:Size label not found

                       

81it [01:41,  1.17s/it]
82it [01:41,  1.20s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #7

                       

82it [01:41,  1.20s/it]
83it [01:43,  1.21s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #6

                       

83it [01:43,  1.21s/it]
84it [01:44,  1.23s/it]INFO:srcopsmetrics.iterator:[API requests remaining: 3319]
INFO:srcopsmetrics.entities.pull_request:Extracting PR #3

                       

84it [01:44,  1.23s/it]
                       

84it [01:44,  1.23s/it]
85it [01:45,  1.26s/it]INFO:srcopsmetrics.entities.pull_request:Extracting PR #2

                       

85it [01:45,  1.26s/it]
86it [01:47,  1.42s/it]
86it [01:47,  1.25s/it]
WARNING:srcopsmetrics.entities.interface:Data found to be inconsistent with its schema, original message:
WARNING:srcopsmetrics.entities.interface:extra keys not allowed @ data['96']['changed_files']
INFO:srcopsmetrics.entities.interface:Knowledge file PullRequest.json
INFO:srcopsmetrics.entities.interface:new 86 entities
INFO:srcopsmetrics.entities.interface:(overall 86 entities)
INFO:srcopsmetrics.entities.interface:Saved on CEPH at thoth/data/thoth-test-core/thoth-sli-metrics/kebechet-update-manager/thoth-station/stub-api/bot_knowledge/thoth-station/stub-api/PullRequest.json
INFO:srcopsmetrics.bot_knowledge:

INFO:srcopsmetrics.entities.tools.storage:Loading knowledge from Ceph
Traceback (most recent call last):
  File "srcopsmetrics/cli.py", line 192, in <module>
    cli(auto_envvar_prefix="MI")
  File "/opt/app-root/lib64/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/opt/app-root/lib64/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/opt/app-root/lib64/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/app-root/lib64/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "srcopsmetrics/cli.py", line 165, in cli
    kebechet_metrics = KebechetMetrics(repository=repos[0], today=True, is_local=is_local)
  File "/opt/app-root/src/srcopsmetrics/kebechet_metrics.py", line 57, in __init__
    self.prs = PullRequest(gh_repo).load_previous_knowledge(is_local=is_local)
  File "/opt/app-root/src/srcopsmetrics/entities/interface.py", line 155, in load_previous_knowledge
    else KnowledgeStorage().load_remotely(self.file_path)
  File "/opt/app-root/src/srcopsmetrics/entities/tools/storage.py", line 136, in load_remotely
    return pd.DataFrame.from_records(data, index="id")
  File "/opt/app-root/lib64/python3.8/site-packages/pandas/core/frame.py", line 1871, in from_records
    i = columns.get_loc(index)
  File "/opt/app-root/lib64/python3.8/site-packages/pandas/core/indexes/range.py", line 354, in get_loc
    raise KeyError(key)
KeyError: 'id'
```

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
Fixes #389 #390 

## This introduces a breaking change

- [ ] Yes
- [X] No

## This Pull Request implements
Make `kebechet_metrics.py` work with dataframes (because of the recent #363 )

### CODE REVIEW

**Comment by xtuchyna** [comment]
Hello @pacospace , could you please have a look at this?

File: srcopsmetrics/kebechet_metrics.py [hunk 1]
```diff
@@ -46,6 +47,18 @@
 _ROOT_DIR = "kebechet-update-manager"
 
 
+def get_update_manager_request_type(title: str) -> Optional[str]:
+    """Get the type of the update request."""
+    if title == "Kebechet update":
```
Comment (pacospace) [inline srcopsmetrics/kebechet_metrics.py:52]: `if title == UPDATE_TYPES_AND_KEYWORDS["manual"]`?

File: srcopsmetrics/kebechet_metrics.py [hunk 2]
```diff
@@ -182,23 +166,27 @@ def get_daily_stats_update_manager(self) -> Dict[str, Any]:
         if prs.empty:
             return {}
 
-        prs["days"] = prs.apply(lambda x: datetime.fromtimestamp(x["date"]).date(), axis=1)
-        today = datetime.now().date()
+        # prs["days"] = prs.apply(lambda x: datetime.fromtimestamp(x["created_at"]).date(), axis=1)
```
Comment (pacospace) [inline srcopsmetrics/kebechet_metrics.py:169]: why comented?

File: srcopsmetrics/kebechet_metrics.py (hunk 1 above)
Comment (xtuchyna) [reply srcopsmetrics/kebechet_metrics.py:52]: nice catch :)

File: srcopsmetrics/kebechet_metrics.py (hunk 2 above)
Comment (xtuchyna) [reply srcopsmetrics/kebechet_metrics.py:169]: forgot to delete, thxx!

**Review by pacospace — APPROVED** [review]
lgtm 💯

**Pull request merged**